src/explore-education-statistics-common/src/components/ExportButtonMenu.module.scss

```diff
@@ -0,0 +1,23 @@
+@import '~govuk-frontend/dist/govuk/base';
+
+.exportDropdown {
```

Comment:
I think the linter will probably object to the css rules not being in alphabetical order.